tests/primer/test_primer_external.py

```diff
@@ -0,0 +1,65 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
+
+import subprocess
+
+import pytest
+
+from pylint.testutils.primer import (
+    PRIMER_DIRECTORY,
+    PackageToLint,
+    clone_primer_packages,
+)
+
+PACKAGES_TO_LINT = {
+    "black": PackageToLint(
+        "https://github.com/psf/black.git", "main", None, "/psf/black", "src tests"
+    ),
+    "home-assistant": PackageToLint(
+        "https://github.com/home-assistant/core.git",
+        "dev",
+        None,
+        "/home-assistant/core",
+        "homeassistant",
+    ),
+    "graph-explorer": PackageToLint(
+        "https://github.com/vimeo/graph-explorer",
+        "master",
+        None,
+        "/vimeo/graph-explorer",
+        "graph_explorer",
+    ),
```

Comment:
```suggestion
        "master",
        None,
        "/vimeo/graph-explorer",
        "graph_explorer",
    ),
    "pygame": PackageToLint(
        " https://github.com/pygame/pygame",
        "main",
        None,
        "/pygame/pygame",
        "src_py",
    ),
```

Reply:
Should we add a `args` attribute to `PackageToLint`? Perhaps we want to disable certain messages or plugins for certain packages in the future? We currently can't do so.

Reply:
654b2fba1ba7fe3b9213c3b33aa86559e5ddebb2